db/primary_migrate/20230502235856_remove_profile_fraud_boolean_columns.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveProfileFraudBooleanColumns < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured {
+      remove_column :profiles, :fraud_review_pending, :boolean
+      remove_column :profiles, :fraud_rejection, :boolean
```

Comment:
It's not strictly required, but we should add the `drop_index` for `index_profiles_on_fraud_review_pending`